pkg/api/controller.go

```diff
@@ -170,10 +175,47 @@ func (c *Controller) Logout(w http.ResponseWriter, _ *http.Request) {
 		Expires:  time.Unix(0, 0),
 		SameSite: http.SameSiteStrictMode,
 	})
-
+	session, err := c.sessionStore.Get(r, OIDCAuthSessionName)
+	if err != nil {
+		writeError(w, http.StatusInternalServerError, err)
+		return
+	}
+	session.Values = nil
+	err = session.Save(r, w)
+	if err != nil {
+		writeError(w, http.StatusInternalServerError, err)
+		return
+	}
```

Comment:
what happens if OIDC is not configured?
shouldn't this return an error?

Reply:
This is the global logout, not specific to OIDC.
We could perform this part only if OIDC is configured - but maybe it's better to clear the session even if OIDC is disabled.